src/components/ui/PreviewProfile.jsx

```diff
@@ -0,0 +1,21 @@
+function PreviewProfile() {
+    return (
+        <div className='container relative :w-full h-full  my-5 flex justify-center'>
+            <div
+                className='absolute bg-cover w-80 h-80 bg-center'
```

Comment:
Add w-full to the profile image to ensure it stretches to the full width of the container

Reply:
let the height of the profile image to h-80 by default and added sm:h-96 and md:h-128 to make it taller on small and medium screens.